themes/aframe/source/css/_settings.styl

```diff
@@ -17,6 +17,7 @@ $code-font-size = 0.8rem
 
 // Colors
 $dark   = #2c3e50
+$fair = lighten(#2c3e50, 20%)
```

Comment:
Ah oops, didn't look closely since I was looking at the PR before it updated.

I meant the orange.

<img width="99" alt="screen shot 2016-05-31 at 12 30 09 pm" src="https://cloud.githubusercontent.com/assets/674727/15687796/6ff48df6-272b-11e6-84e8-1efdc026a6a6.png">